pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -318,40 +318,40 @@ def test_dt64arr_timestamp_equality(self, box_with_array):
             box_with_array if box_with_array not in [pd.Index, pd.array] else np.ndarray
         )
 
-        ser = Series([Timestamp("2000-01-29 01:59:00"), "NaT"])
+        ser = Series([Timestamp("2000-01-29 01:59:00"), Timestamp("2000-01-30"), "NaT"])
```

Comment:
why is this necessary?

Reply:
Because I made the dataframe (2, N) shaped, a series with 2 elements would give a (2, 2) dataframe (and a column would be length 2), which would no longer need alignment, changing the test (eg it no longer would warn).